subsys/bluetooth/audio/has.c

```diff
@@ -1122,12 +1132,23 @@ int bt_has_preset_name_change(uint8_t index, const char *name)
 }
 #endif /* CONFIG_BT_HAS_PRESET_SUPPORT */
 
-static int has_init(const struct device *dev)
+int bt_has_register(const struct bt_has_register_param *param)
 {
-	ARG_UNUSED(dev);
+	LOG_DBG("Trying to start HAS service...");
+	int err;
+
+	CHECKIF(!param) {
+		LOG_DBG("NULL params pointer");
```

Comment:
I think that we should use BT_DBG here instead

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/52672?notification_referrer_id=NT_kwDOACZ1XbI0OTYyMjkzMzc4OjI1MjA0MTM#discussion_r1036895465